.github/actions/helm-lint/action.yml

```diff
@@ -22,6 +22,6 @@ runs:
         set -o pipefail
 
         helm dependency update .
-        # produces `helm lint . --with-subcharts -f file1 -f file2` with values_files as file1,file2
-        helm lint . --with-subcharts $(echo "${{ inputs.values_files }}" | sed 's/,/ -f /g' | sed 's/^/-f /')
+        # produces `helm lint . -f file1 -f file2` with values_files as file1,file2
```

Comment:
what happens if the chart as dependencies?

Reply:
without `--with-subcharts` the `helm lint` command still detects errors when i run this on the example app